src/main/include/subsystems/DriveSubsystem.h

```diff
@@ -37,14 +38,22 @@ struct Encoders {
 
 class DriveSubsystem : public frc2::SubsystemBase {
    public:
+        WPI_TalonFX& RightLead;
+        WPI_TalonFX& LeftLead;
```

Comment:
Make private again and expose only the required methods that need to be called from them